### ISSUE TITLE
Add accessible content to Keyboard Shortcuts in version with Grab button

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#138]
@zepumph, likely already on your radar. Adding issue just to be sure.

Before next interviews (maybe October 26th) the Grab button version will need accessible content in the dialog and the dialog with need some extra content (taking screenshot from BASE)

The dialog will need to describe the two step grab then move interaction.
<img width="392" alt="base-grab-instructions" src="https://user-images.githubusercontent.com/1902116/47221776-9eb4f200-d3b5-11e8-9090-63c4b911ab45.png">

So in Friction this means the content dialog headings will need to read like this:
[...]
**H2: Grab or Release Book or Zoomed-in Book**
P: Grab or release book [Space] or [Enter]

**H2: Move Book or Zoomed-in Book**
_Then the rest is the same content as you have for the non-grab-button version._

I've put the accessible content [here in the design document](https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.b21tp86ot9um), the main difference is generally only the addition of "with" before [keyboard button icons].

**Comment by zepumph** [comment]
I added it. For some reason the spacing is a bit weird visually, but the content should be solid. I will look into the spacing, perhaps with @jessegreenberg if I can't figure it out. For now @terracoda does everything else look good?

**Comment by terracoda** [comment]
@zepumph, apologies, I had some typos in the design document which I have fixed (see read text) 

Posting the corrected text is here, too:

H2: Move Book or Zoomed-in Book
UL
  LI: Move grabbed book up, left, down, or right with Arrow keys, or with letter keys W, A, S, or D
  LI: Move slower with Shift plus Arrow keys, or Shift plus letter keys W, A, S, or D

**Comment by zepumph** [comment]
Implemented above. I'm going to close this. @terracoda please reopen if there is more, or create a new issue.

**Comment by terracoda** [comment]
@zepumph, here's the latest visual
<img width="881" alt="friction-kbshortcuts" src="https://user-images.githubusercontent.com/1902116/48417958-5db1c280-e754-11e8-993f-1bf355174fda.png">

Under the Grab and Release heading, we just need:

> Grab or release book [Space] or [Enter]

No need to repeat "zoomed-in book".

**Comment by zepumph** [comment]
How's that?

**Comment by terracoda** [comment]
@zepumph, looks good, but needs a little pixel polishing

**1 change in visual dialog:**
- adjust heading size

**1 change in a11y dialog:**
- remove "the" before "book" in the accessible content

**Comment by zepumph** [comment]
I'm not sure how to go about adjusting the heading. It was scaled down
because of it's length, in order to fit in the dialog. I'll take a look and
see whay our best options are.

**Comment by terracoda** [comment]
@zepumph considering your comment about the automatic heading scaling, would it be more straight forward to keep the heading content simple and adjust the paragraph text and list item text to accommodate "the zoomed-in book"? I was thinking there was more room in the headings, but maybe not.

**Try this as a work around:**

**H2: Grab or Release Book**
- P: Grab or release book or zoomed-in book [Space] or [Enter]
  - A11y Content: Grab or release book or zoomed-in book with Space or Enter keys.

**H2: Move Book**
- LI: Move grabbed book or zoomed-in book [up][left][down][right] or [W][A][S][D]
  - LI: Move grabbed book or zoomed-in book up, left, down, or right with Arrow keys, or with letter keys W, A, S, or D.
- LI: Move slower [Shift] + [up][left][down][right] or [Shift] + [W][A][S][D] (stacked)
  - A11y content: Move slower with Shift plus Arrow keys, or Shift plus letter keys W, A, S, or D.

@zepumph, just noticed that "Move in smaller steps" should be "Move slower". Please change to "Move slower" in both the visual and non-visual view.

**Comment by zepumph** [comment]
Implemented; here is one note. When we add "grabbed" to LI: "Move [grabbed] book or zoomed-in book", notice that we have the same scaling issues but now for the LI. If "grabbed" was omitted, then it would visually be the same.

@terracoda please review.

**Comment by terracoda** [comment]
@zepumph, I see the problem. The body text is now scaled smaller:
<img width="770" alt="friction-badly-scaled-text" src="https://user-images.githubusercontent.com/1902116/49014312-03ffbe00-f180-11e8-8dda-6e3f03f67dda.png">


Is is possible to wrap the body text instead of scaling it? I wouldn't want to wrap heading text, of course. Is something like this (below) possible? If so, I can ask a designer (maybe @arouinfar) to do a proper polished mock-up for you, if needed. The mock-up below is very rough.
<img width="759" alt="rough-draft-wrapped-text" src="https://user-images.githubusercontent.com/1902116/49014332-0f52e980-f180-11e8-9aa2-5ffd8f92cfe6.png">

**Comment by terracoda** [comment]
@zepumph we can discuss in today's design meeting.

**Comment by zepumph** [comment]
We discussed today, and I wrapped the text in the above commit. How does this look @terracoda? 
![image](https://user-images.githubusercontent.com/6856943/49109293-cc078200-f236-11e8-835e-79a8ba311ff0.png)

**Comment by terracoda** [comment]
@zepumph, the screenshot above looks great, but I see something slightly different where the buttons are not aligned due to the second "grabbed" in the second check of text...
<img width="825" alt="screen shot 2018-11-28 at 8 03 00 am" src="https://user-images.githubusercontent.com/1902116/49135070-f3695800-f2e5-11e8-9113-dec2768ba3aa.png">

Did something change? Or this perhaps browser related? I tested in FF and Safari and I see the same thing in both.

**Comment by terracoda** [comment]
@zepumph, oh, I see your screenshot doesn't have the second "grabbed". If possible, please tweak the wrapping a little more so the wrap happens after the "or" in both cases, or if not possible, just remove the second explicit "grabbed" before "zoomed-in book".

**Comment by arouinfar** [comment]
@zepumph the BASE dialog has a similar structure (sans wrapped text) and might be helpful in figuring out how to align the keys in the left column.

![image](https://user-images.githubusercontent.com/8419308/49408313-ff577e80-f718-11e8-8a95-5bb7b1accb5f.png)

If wrapping the text somehow breaks the ability to left-align the key icons, then https://github.com/phetsims/friction/issues/138#issuecomment-443909254 doesn't look so bad to me. :)

**Comment by terracoda** [comment]
@zepumph, I agree with @arouinfar #138 looks pretty good, but if possible move the [Space] or [Enter] buttons to the right just a smudge to align vertically with the first [up] arrow button.

**Comment by zepumph** [comment]
Thank you both for the assistance. I looked at BASE and found the line of code I needed. Should now be aligned.

![image](https://user-images.githubusercontent.com/6856943/49413022-067f8c00-f71b-11e8-8331-d062672a294d.png)

**Comment by terracoda** [comment]
@zepumph, that is beautiful alignment.

In the A11y View, I noticed we are missing a two periods.

The bullets about moving the grabbed book and grabbed zoomed-in book each need a period at the end.

I checked the Keyboard Shortcuts dialogs for BASE and RIAW to double check for consistency. We do, indeed, have periods in the other sims' dialogs.

The "Tab to get started" should have a period, too, but that's in common code, so I can make a separate issue for that period.

<img width="459" alt="friction" src="https://user-images.githubusercontent.com/1902116/49434540-ad643680-f7b4-11e8-9c04-c528fbba3a70.png">

Once the periods are added, that's a wrap!

**Comment by zepumph** [comment]
Periods added. Closing

**Comment by terracoda** [comment]
@zepumph, thanks for adding the periods!
